README.md

```diff
@@ -1,90 +1,141 @@
+---
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
```

Comment:
This looks strange on GitHub. Why do we need this?
![grafik](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/assets/25061876/ba6c4daa-8f72-4b86-a81d-aad69c2b6c13)

Reply:
lol, this was added by RStudio markdown editor.. will remove.